index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Habby's Page</title>
+	<link rel="stylesheet" type="text/css" href="style.css" >
+	<meta charset="UTF-8">
+
+</head>
+<body>
+
+<img src="http://i.imgur.com/enRYXYj.jpg?1" alt="HabbyPic" title="HabbyPic">
+<section>
+	<h1>About Me...</h1>
+	<p>Hi there, my name is Habby and I have no idea how I ended up here (meaning the world of technology) but geesh!...what took me so long?  I have always had an affinity for all things creative, but I just didn't know how to apply it until it just hit me!
```

Comment:
Because HTML ignores line breaks, I'd use line breaks to keep these paragraphs from getting too wide in the text editor. Again, this won't impact how it displays in the browser, so no reason not to make your HTML more readable!

```html
<p>Hi there, my name is Habby and I have no idea how I ended up here 
(meaning the world of technology) but geesh!...what took me so long?  
I have always had an affinity for all things creative, but I just didn't know 
how to apply it until it just hit me!</p>
```